src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -85,26 +85,24 @@ const FaultEventMenu = ({ selectedShapeToolData, onEventUpdated, refreshTree, ro
   const [failureModeOverviewDialogOpen, setFailureModeOverviewDialogOpen] = useState(false);
   const [failureModeOverview, setFailureModeOverview] = useState<FailureMode | null>(null);
 
-  const [criticality, setCriticality] = useState<number | undefined>(undefined);
-  const [predictedFailureRate, setPredictedFailureRate] = useState<number | undefined>(undefined);
-  const [ataSystem, setAtaSystem] = useState<string | undefined>(undefined);
-  const [partNumber, setPartNumber] = useState<string | undefined>(undefined);
-  const [stock, setStock] = useState<string | undefined>(undefined);
-  const [quantity, setQuantity] = useState<number | undefined>(undefined);
-  const [schematicDesignation, setSchematicDesignation] = useState<string | undefined>(undefined);
+  const [criticality, setCriticality] = useState<number | undefined>(0);
+  const [predictedFailureRate, setPredictedFailureRate] = useState<number | undefined>(0);
```

Comment:
Could you explain me why there are in this file both:
`setPredictedFailureRate` and `setSnsPredictedFailureRate` -- is it same or first one should be somehow renamed so it is not that confusing?